topology/probes/netlink.go

```diff
@@ -373,11 +373,14 @@ Loop:
 	for {
 		n, err := syscall.EpollWait(epfd, events[:], 1000)
 		if err != nil {
-			logging.GetLogger().Error("Failed to receive from netlink messages: %s", err.Error())
-			continue
+			errno, ok := err.(syscall.Errno)
+			if ok && errno != syscall.EINTR {
```

Comment:
you should not do like this, 
syscall.EINTR **!=** Ctrl+C (SIGINT)
A Syscall can return EINTR for many other reasons, like after attach gdb to the process and continue, basically it's a cancellation point.

BTW, the chan is used as a asynchrous flag and break Loop didn't break the for loop.
I will annotate the code we a solution